File: PruebaTecnicaDeloimBackend/PruebaTecnicaDeloimBackend.Api/Models/AfiliadoModel.cs
using System;

namespace PruebaTecnicaDeloimBackend.Api.Models
{
    public class AfiliadoModel
    {
        public int AfiliadoID { get; set; }
        public string AfiliadoNombre { get; set; }
        public DateTime? FechaNacimiento { get; set; }
        public string LugarNacimiento { get; set; }
        public byte[] Foto { get; set; }
        public decimal Peso { get; set; }
    }
}

File: PruebaTecnicaDeloimFrontend/src/app/afiliado/models/afiliado.model.ts
import { AfiliadoData } from '../interfaces/afiliado.interface';

export class Afiliado implements AfiliadoData {

  constructor(
    public afiliadoID:  number,
    public afiliadoNombre:  string,
    public fechaNacimiento: Date,
    public lugarNacimiento: string,
    public foto: string,
    public fotoBase64: string,
    public peso: number,
    ) {}
}
